_data/releases.yaml

```diff
@@ -695,3 +695,7 @@ bridge:
   - version: 0.12.2
   - version: 0.12.1
   - version: 0.12.0
+drainCleaner:
+  - version: 1.2.0
+accessOperator:
+  - version: 0.1.0
```

Comment:
I think we have `0.1.1` released

Reply:
We do have 0.1.1 out. But the Operator 0.45 docs and installation file have 0.1.0 in them as this is something we update only with next Operator release. So I guess this is a bit confusing 🤔. Not sure what is the best approach here. Should we have the link without mentioning any exact version and have the exact version in downloads?

Reply:
Yeah sorry, I didn't realize that we have the `0.1.0` in the installation files 🙂 
But as you mentioned, I would maybe keep it without the version. For Bridge, we have a separate section just for it, right? That's why it's okay to keep the version there, even if the version is not part of the Operators installation files.

Reply:
I understand that Bridge can run against any Kafka, so it makes sense to me that it would have it's own link. I would propose a different approach. What if instead of adding links to Drain Cleaner and Access Operator here, which are only useful if you are using the Cluster Operator, we made them more prominent in the Cluster Operator docs pages. So e.g. mention them specifically in the overview and look at how they can be more easily found in the deploying and upgrading page?

Reply:
Yes, the bridge can run against any Kafka and it's not relegated to Kubernetes but also bare metal and VMs so it deserves its own link imho. Regarding drain cleaner and access operator, finding a way to highlight them in the doc is not that simple, because they could be easily lost but maybe it's a good approach.

Reply:
Where would you add this piece?

Reply:
On the Documentation page. Above the guide links.